src/hooks/gql.ts

```diff
@@ -37,6 +41,33 @@ function useTypedQuery<
   );
 }
 
+// this isn't used yet, but it would be great to be able to use it
+// I couldn't figure out how to get it to work with variables -CryptoGraffe
```

Comment:
@META-DREAMER @topocount maybe can help get the variables to work here.

Reply:
I think we should put a pin in this and attack this issue in separate task soon after this one. To be clear, I don't want to deprioritize this for @CryptoGraffe, I just think it should be a separate PR that closely follows this one.

There's already a lot going on in this one.

Reply:
I'm content leaving this code in here and addressing it as we will out the FE later.